### DISCUSSION TITLE
Will i get any ban on my discord account?

### DISCUSSION THREAD

**Discussion by tommytheturtle1** [victornpb/undiscord#453] [1. Questions & Answers] [answered]
Being rate limited by the API, is this message any threat to my discord account being deleted or terminated cause i am deleting too many messages? I just want to know the consequences if there are any

My current delay is 2100 and 2100, do you any have suggestions to improve the time?

**Comment by twardy95** [comment, marked as answer]
Being rate limited is normal and its only there to ensure the user doesnt send too many requests in a span of time. It can be triggered if you for example spam send messages or do things too fast in normal cases. So the API doesnt necessarily know if youre using a bot or not. Meaning that you should feel safe.

I usually use either 500 ms or 1000 ms on message deletion delay and 100 ms on message search and let it run. It adjusts itself to around 1000 ms after a while because of being rate limited but it does the job quite fast. 

I heard people getting banned but usually after a couple of years or months. Havent gotten banned yet and ive been using it for over a week now.

Hope this clears up some things.

**Comment by tommytheturtle1** [comment]
Thanks and how many messages have you deleted so far over a week?

**Reply by twardy95** [reply]
Over 80k messages. It usually takes an hour to delete around 1-2k messages with 1000 ms delay.
